doc/development_guide/contributor_guide/contribute.rst

```diff
@@ -13,9 +13,9 @@ your patch gets accepted:
   you access to the latest ``ast`` parser.
 - Install the dev dependencies, see :ref:`contributor_install`.
 - Use our test suite and write new tests, see :ref:`contributor_testing`.
-- Add an entry to the change log describing the change in `doc/whatsnew/2/2.15/index.rst`
-  (or ``doc/whatsnew/2/2.14/full.rst`` if the change needs backporting in 2.14).
-  If necessary you can write details or offer examples on how the new change is supposed to work.
+- Create a news fragment with `towncrier create <IssueNumber>.<type>` which will be
+  included in the changelog. If necessary you can write details of offer examples on how
+  the new change is supposed to work.
```

Comment:
I tried the following:
```rst
.. include:: ../../../.github/PULL_REQUEST_TEMPLATE.md
   :start-after: - [ ] Write a good description on what the PR does.
   :end-before: - [ ] If you used multiple emails or multiple names when contributing, add your mails
```

However, I did not find an option that would allow me to drop the `[ ]` checkbox we have in the PULL_REQUEST_TEAMPLATE.md.
Any idea how to do that?

Reply:
Maybe we can include the newline and `- [ ]` in the start after ?

Reply:
I tried that, but simply appending "\n - [ ]" did not work, and Sphinx complained that it could not find the string.

Reply:
I think you need a litteral newline not `\n` . Let's simply add a comment to remember that it's a duplication if it does not work.